src/test/java/com/exasol/cloudetl/kafka/KafkaTestSetup.java

```diff
@@ -53,37 +63,43 @@ static KafkaTestSetup create() throws ExecutionException, InterruptedException {
             final CreateTopicsResult result = admin.createTopics(Collections.singleton(newTopic));
 
             final KafkaFuture<Void> future = result.values().get(topicName);
-            final var createTopicsResult = future.get();
+            future.get();
+
+            //final Set<String> topicNames = admin.listTopics().names().get();
+
+            LOG.info("Succesfully created topic");
         } catch (final Exception ex) {
             LOG.warning("Exception occurred during Kafka topic creation: '" + ex.getMessage() + "'");
+            throw ex;
         }
+        // PRODUCE
```

Comment:
Extract the code to produce entries into a separate method and call it from the test when running the test.
Else only these two entries are produced and when additional tests (e.g. upgrade test) also read entries they won't find any.

Reply:
done